home/src/main/resources/config/example.runtime.properties

```diff
@@ -196,8 +196,8 @@ proxy.eligibleTypeList = http://www.w3.org/2002/07/owl#Thing
 #fileUpload.allowedMIMETypes = image/png, application/pdf
 
 # Feature toggle for forgot password functionality
-authentication.forgotPassword = enabled
-authentication.forgotPassword.notify-admin = true
+authentication.forgotPassword = false
```

Comment:
please change the value to disabled (instead of false), to make it aligned with Java code.